CONTRIBUTING.md

```diff
@@ -65,41 +65,59 @@ Steps to get started:
 
 ## Getting your development environment set up
 
-After cloning the project there are a few steps required to get the project running.
+After cloning the project there are a few steps required to get the project running. 
+For running all tests, take the following steps:
 
-1.  Fetch submodules to pull ECMAScript test suites.
+1. Fetch submodules to pull ECMAScript test suites.
 
     ```bash
     git submodule update --init --recursive
     ```
 
-2.  Install js dependencies.
+2. Install js dependencies.
 
     ```bash
     yarn
     ( cd crates/swc_ecma_transforms; yarn )
```

Comment:
I think this is no longer required.

Reply:
Oh, you are right.

Reply:
I remember we need run yarn at root folder to install mocha,
and setup a path to node_modules/.bin

Reply:
what should we change this to? no longer needed, or different command. 

Should also call out yarn as a dependency probably.

Reply:
No longer needed, but `yarn` in root directory is required.